### ISSUE TITLE
how to have a isPlaying status like ?

### ISSUE THREAD

**Issue by jeremy-habit** [hyochan/react-native-audio-recorder-player#365]
### Version of react-native-audio-recorder-player
3.1.0

### Version of React Native
0.64.1
### Platforms you faced the error (IOS or Android or both?)
ios 
### Expected behavior
do not receive a last event from playback listener after pause
### Actual behavior
when i use onPausePlay, it's working BUT after pausing action i receive a last event from the playback listener. When i pause, i setState isplaying at false, and isPasued at true. But in the callback from playbacklisten i'm setting isPlaying at true, and isPaused at false. And if i received one event of playing after paused, then isPlaying is true even if i used onPausePlay.

**Comment by hyochan** [comment]
Yeah looks like a race condition. I am not currently sure how to make this perfectly.

**Comment by efstathiosntonas** [comment]
same here, after i pause the player, a last event of play is being sent back on ios.

**Comment by hyochan** [comment]
This issue was filed against the pre-Nitro `react-native-audio-recorder-player` API. This repo has since been completely rewritten on top of [react-native-nitro-modules](https://github.com/mrousavy/nitro) as **react-native-nitro-sound**.

If this still reproduces on the latest `react-native-nitro-sound`, please open a fresh issue with:
- Library version (`react-native-nitro-sound`)
- React Native version
- Platform (iOS / Android)
- Minimal repro steps and logs

Closing as `stale-legacy` for now.

> 🤖 AI-native maintenance. See #787 for context.

**Comment by hyochan** [comment]
Closing as part of a stale-legacy sweep.

This issue was filed against the pre-Nitro `react-native-audio-recorder-player` codebase, hasn't seen activity in a long time, and the underlying API has since been rewritten on top of [Nitro Modules](https://github.com/mrousavy/nitro). Most legacy code paths referenced here no longer exist.

If this still affects you on **`react-native-nitro-sound` v3+**, please:
1. Open a fresh issue with a minimal repro on the current version, and
2. Include your iOS/Android version, RN version, and `nitro-modules` version.

Thanks for the original report — closing to keep the queue focused on actionable, current-version reports. 🙏

> 🤖 Auto-closed by daily triage.
